addons/survey/controllers/main.py

```diff
@@ -472,6 +472,50 @@ def survey_begin(self, survey_token, answer_token, **post):
         answer_sudo._mark_in_progress()
         return self._prepare_question_html(survey_sudo, answer_sudo, **post)
 
+    @http.route('/survey/<string:survey_token>/<string:answer_token>/check_answers', type='json', auth='public', website=True)
+    def survey_check_answers(self, survey_token, answer_token, **post):
```

Comment:
As you sometimes return a dictionary, including for the error cases, other times `False`, could you add a small docstring?